src/index.css

```diff
@@ -0,0 +1,71 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+  font-family: "Mont";
+  src: url("/fonts/Mont-Regular.otf") format("opentype");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Mont SemiBold";
+  src: url("/fonts/Mont-SemiBold.otf") format("opentype");
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Mont Bold";
+  src: url("/fonts/Mont-Bold.otf") format("opentype");
+  font-weight: 800;
+  font-style: normal;
+}
+
+body,
+html {
+  overflow-x: hidden;
+  width: 100dvw;
+}
+
+#root {
+  min-height: 100dvh;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+}
```

Comment:
I think problem lies not with banner but with this grid. You can try to fix it or completely delete it and then adjust the rest

Reply:
Grid was not a problem, but solution =) It fix my layout in good way and stopped change page wigth